sdk/test-utils/recorder-new/src/utils/env.ts

```diff
@@ -6,4 +6,4 @@ import * as dotenv from "dotenv";
 // Initialize the environment
 dotenv.config();
 
-export const env = process.env;
+export const env = process.env as Record<string, string | undefined>;
```

Comment:
This should more-or-less be the definition of `process.env` already. `NodeJS.Dict<string>` which should expand to `{ [k: string]: string | undefined }`.